assets/src/edit-story/components/panels/panel/content.js

```diff
@@ -33,6 +33,8 @@ import panelContext from './context';
 const Form = styled.form`
   padding: ${({ padding }) => padding || '10px 20px'};
   overflow: auto;
+  border: 0 solid ${({ theme }) => theme.colors.fg.v6};
+  border-top-width: ${({ hasBorder }) => (hasBorder ? '1px' : '0')};
```

Comment:
Why not something like this:

```js
${({ hasBorder, theme }) => hasBorder && `border-top: 1px solid ${theme.colors.fg.v6};`}
```

Reply:
Fixed in 19fad9df (with a bonus of removing another rule, that we actually don't need as it is browser native behaviour).